prototypes/entity.lua

```diff
@@ -96,40 +125,172 @@ data:extend({
+return
+	{
+		layers = {
+			{
+				filename = img,
+				width = width/2,
+				height = height/2,
+				repeat_count = 7,
+				hr_version = {
+					filename = hrimg,
+					width = width,
+					height = height,
+					repeat_count = 7,
+					scale = 0.5,
+				}
+			},
+			{
+				filename = chestanim,
+				width = chestanimsize/2,
+				height = chestanimsize/2,
+				frame_count = 7,
+				shift = chestshift,
+				hr_version = {
+					filename = hrchestanim,
+					width = chestanimsize,
+					height = chestanimsize,
+					frame_count = 7,
+					shift = chestshift,
+					scale = 0.5,
+				}
+			},
+			{
+				filename = shadow,
+				width = width/2,
+				height = height/2,
+				shift = shadowshift,
+				repeat_count = 7,
+				draw_as_shadow = true,
+				hr_version = {
+					filename = hrshadow,
+					width = width,
+					height = height,
+					shift = shadowshift,
+					scale = 0.5,
+					repeat_count = 7,
+					draw_as_shadow = true,
+				}
+			},
+		},
+	}
+end
+
+function connectorSprite(shift, shiftshadow)
+return
+	{
+		connector_main =
+		{
+			filename = ENTITYPATH.."connector.png",
+			width = 42,
+			height = 68,
+			shift = shift,
+			hr_version = {
+				filename = ENTITYPATH.."hr-connector.png",
+				width = 84,
+				height = 136,
+				scale = 0.5,
+				shift = shift,
+			}
+		},
+		connector_shadow =
+		{
+			filename = ENTITYPATH.."connector-shadow.png",
+			width = 34,
+			height = 17,
+			draw_as_shadow = true,
+			shift = shiftshadow,
+			hr_version = {
+				filename = ENTITYPATH.."hr-connector-shadow.png",
+				width = 68,
+				height = 33,
+				scale = 0.5,
+				draw_as_shadow = true,
+				shift = shiftshadow,
+			}
+		},
+		led_blue = { filename = "__core__/graphics/empty.png", size = 1 },
+		led_green = { filename = "__core__/graphics/empty.png", size = 1 },
+		led_red = { filename = "__core__/graphics/empty.png", size = 1 },
+		led_light = { type = "basic", intensity = 0, size = 0 }
+	}
+end
+
 -- generate logistic variants
 function createLogisticContainer(name, logistic_type)
 	local p = table.deepcopy(data.raw["container"][name.."-basic"])
 	p.name = name.."-"..logistic_type
+	local img = ENTITYPATH..name.."/"..p.name..".png"
+	local hrimg = ENTITYPATH..name.."/hr-"..p.name..".png"
+	local shadow = ENTITYPATH..name.."/"..name.."-shadow.png"
+	local hrshadow = ENTITYPATH..name.."/hr-"..name.."-shadow.png"
+	local chestanim = ENTITYPATH..name.."/"..name.."-chest-anim.png"
+	local hrchestanim = ENTITYPATH..name.."/hr-"..name.."-chest-anim.png"
 	p.minable.result = p.name
-	p.icon = "__Warehousing__/graphics/icons/"..p.name..".png"
-	p.picture.filename = "__Warehousing__/graphics/entity/"..p.name..".png"
+	p.icon = ICONPATH..p.name..".png"
+	--p.picture.layers[1].filename = img
+	--p.picture.layers[1].hr_version.filename = hrimg
+	--p.picture.layers[2].filename = shadow
+	--p.picture.layers[2].hr_version.filename = hrshadow
```

Comment:
Debug comments? If I'm not lazy I'll remove them, but probably only if I have to go tweak something else.

Reply:
A remnant from a previous version, until I created an animation instead (to have opening chests)